packages/manager/src/factories/account.ts

```
@@ -45,7 +45,7 @@ export const accountFactory = Factory.Sync.makeFactory<Account>({
     'Linodes',
     'LKE HA Control Planes',
     'Machine Images',
-    'Managed Databases V2',
```

Reverting mock data due to causing e2e failure